py/desispec/test/test_proc_night.py

```diff
@@ -13,28 +13,46 @@
 
 import numpy as np
 
-from desispec.workflow.tableio import load_table
+from desispec.workflow.tableio import load_table, write_table
 from desispec.workflow.redshifts import get_ztile_script_pathname
 from desispec.workflow.desi_proc_funcs import get_desi_proc_tilenight_batch_file_pathname
+from desispec.workflow.exptable import get_exposure_table_pathname
 from desispec.io import findfile
+from desispec.test.util import link_rawdata
 
 from desispec.scripts.proc_night import proc_night
+import desispec.scripts.tile_redshifts
 from desiutil.log import get_logger
 
+## directory with real raw data for testing at NERSC
+_dailynight = 20230915
```

Comment:
Agreed.  I had semi-randomly picked a night (N+1 from the other night used for testing non-daily mode), which highlighted that daily mode was more restrictive than it needed to be so I updated the code rather than switching testing nights.  The difference isn't needed now, but might be helpful in the future.

Reply:
I still argue that each has pros and cons, as each is an assumption about how future flat sequences might be ordered and what we will use for cte correction in the future. 
`--daily` mode worked even on historic data. It was only when `--daily` was run _with_ the newly added testing flag `--still-acquiring` on a historic data that doesn't match the format of the current and future flat sequences that there was an issue.
That said, we will use the 1s flat check, which allows this unit test to be run on the night given here.